src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        Lottery ball2 = new Lottery();
+        Lottery ball3 = new Lottery();
+
+        System.out.println(ball1.getRandomBall(ball1));
+        System.out.println(ball2.getRandomBall(ball2));
+        System.out.println(ball3.getRandomBall(ball3));
```

Comment:
There's no need to pass the `Lottery` instance to its own method `getRandomBall`. If `getRandomBall` is an instance method, it can directly access the instance's properties without needing it as a parameter.